pkg/testutil/util.go

```diff
@@ -79,11 +79,18 @@ func IsYamlEqual(expectation *string, actualYamlByteArray *[]byte) bool {
 	}
 
 	// Replace Timestamps that would show up as different.
-	_, err := exec.Command("sed", "-r", "-i", ReplaceTimestampRegExp, actualYamlFileName).Output()
+	_, err := exec.Command("sed", "-r", "-i.tmp", ReplaceTimestampRegExp, actualYamlFileName).Output()
+	// Needed for the ApplicationAutoscaling unit tests to ignore the lastModifiedTime
+	_, err = exec.Command("sed", "-r", "-i.tmp", "/lastModifiedTime/d", actualYamlFileName).Output()
```

Comment:
We should be careful with executing commands like this, it all depend on the OS used. I'm pretty sure the sed binary in Linux is not similar to the one in Darwin systems.

Reply:
This is a good point and we have already run into scenarios where this mismatch has caused failures on my personal mac machine vs Ryan's. We have fixed the issue for now but we plan a larger cleanup of this code in the next iteration.